src/core/p5.Renderer.js

```diff
@@ -298,15 +298,12 @@ p5.Renderer.prototype.text = function(str, x, y, maxWidth, maxHeight) {
       }
     } else {
       // no text-height specified, show warning for BOTTOM / CENTER
-      if (this._textBaseline === constants.BOTTOM) {
-        return console.warn(
-          'textAlign(*, BOTTOM) requires x, y, width and height'
-        );
-      }
-      if (this._textBaseline === constants.CENTER) {
-        return console.warn(
-          'textAlign(*, CENTER) requires x, y, width and height'
-        );
+      if (this._textBaseline === constants.BOTTOM ||
+        this._textBaseline === constants.CENTER) {
+        // use rectHeight as an approximation for text height
+        let rectHeight = p.textSize() * (this._textLeading);
```

Comment:
One last tiny change, we can take out the brackets around `this._textLeading` now 🙂